### PR TITLE
OSD-12049 Ensure openshift-cluster-version is a managed namespace

### PR DESCRIPTION
**Pull request by rendhalver**
This is to ensure the ClusterOperator* alerts get sent to PD

### What type of PR is this?
bug

### What this PR does / why we need it?
ClusterOperator* alerts were not being sent to pagerduty because openshift-cluster-version was not marked as a managed namespace.

### Which Jira/Github issue(s) this PR fixes?

[OSD-12049](https://issues.redhat.com//browse/OSD-12049)

### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [ ] Included documentation changes with PR
- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

**Comment by rendhalver** [comment]
We will be doing some testing with this before we merge
/hold

File: deploy/osd-managed-resources/managed-namespaces.ConfigMap.yaml
Comment (jewzaam) [inline deploy/osd-managed-resources/managed-namespaces.ConfigMap.yaml:22]: This is a creep in scope and not something included in OSD-12049.  This should be removed and addressed in a separate ticket if desired.

**Review by jewzaam — CHANGES_REQUESTED** [review]

File: deploy/osd-managed-resources/managed-namespaces.ConfigMap.yaml
Comment (rendhalver) [reply deploy/osd-managed-resources/managed-namespaces.ConfigMap.yaml]: Manually removed for now.
OSD-12049 has been created to discuss that.

File: resources/managed/all-osd-resources.yaml
Comment (cblecker) [inline resources/managed/all-osd-resources.yaml:55]:
```suggestion
```

File: resources/managed/all-osd-resources.yaml
Comment (cblecker) [inline resources/managed/all-osd-resources.yaml:346]:
```suggestion
```

File: resources/managed/all-osd-resources.yaml
Comment (cblecker) [inline resources/managed/all-osd-resources.yaml:436]:
```suggestion
```

**Review by cblecker — DISMISSED** [review]

**Pull request merged**
